app/views/events/index.slim

```diff
@@ -1,11 +1,13 @@
-h2 Events
--@events.each do |event|
-  = link_to event.name, event_path(event.id)
-  ul
-    li 
-      b = "On #{Date::DAYNAMES[event.start_date.wday]}"
-    li = event.starting_date
-    li = event.ending_date
-      li.fa.fa-user = "#{event.attendees.size}"
--if !current_user.nil?
-  = link_to 'New Event', new_event_path
+ul.Grid
+  -@events.each do |event|
+    .Grid-item
+      li.Event
+        = link_to event_path(event.id), class: 'Event-title' do
+          .Event-title = event.name
+          .Event-info
+            em = "On #{Date::DAYNAMES[event.start_date.wday]}"
+          .Event-info = event.starting_date
```

Comment:
The second line wasn't generating anything.
The idea was to present a user icon followed by the number of attendees.
I came up with 
```
.Event-info 
  span.fa.fa-user = "#{event.attendees.size}"
```